rules/early-return/tests/Rector/If_/ChangeOrIfReturnToEarlyReturnRector/Fixture/rex_formatter.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Rector\EarlyReturn\Tests\Rector\If_\ChangeOrIfReturnToEarlyReturnRector\Fixture;
+
+abstract class rex_formatter
+{
+    private static function getTimestamp($value)
+    {
+        if (is_int($value) || ctype_digit($value)) {
+            return (int) $value;
+        }
+
+        return strtotime($value);
+    }
+}
+?>
+-----
+<?php
+namespace Rector\EarlyReturn\Tests\Rector\If_\ChangeOrIfReturnToEarlyReturnRector\Fixture;
+
+abstract class rex_formatter
+{
+    private static function getTimestamp($value)
+    {
+        if (is_int($value)) {
+            return $value;
```

Comment:
as is, rector inserts a `(int)` case here, which is redundat, regarding the `if` expression was transformed into `is_int()`

Reply:
The code example itself already always redundant cast to `(int)`:

```php
        if (is_int($value) || ctype_digit($value)) {
            return (int) $value;
        }
```

the rule is about use the statement in if statement, not changing casting.